url/README.md

```diff
@@ -1,6 +1,19 @@
-The test for browsers is `a-element.html`. The reusable format is `urltestdata.txt`, which
-is not documented in detail. Reverse engineering through `urltestparser.js` should not be
-too hard. Documentation welcome!
+The test for browsers is `a-element.html`. The reusable format is `urltestdata.json`,
+which contains a JSON array of comments as strings and test cases as objects.
+The keys for each test case are:
+
+* `base`: an absolute URL as a string whose [parsing] without a base of its own should succeed.
```

Comment:
Maybe document that URLs are always parsed with a base; "absolute URLs" are parsed against "about:blank"